frontend/public/index.html

```diff
@@ -16,10 +16,6 @@
     />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
-      rel="stylesheet"
-    />
```

Comment:
This PR also has font related changes: https://github.com/CollActionteam/clothing-loop/pull/128

Maybe coordinate with @giuliamml , because it seems you guys have some conflicts between those two branches.

Reply:
This seems an easier way to import the font in the project so can approve these changes. I will review and make changes to my PR accordingly

Reply:
Thank you for the review :)